requirements.txt

```diff
@@ -1,8 +1,10 @@
 batinfo==0.3
+git+https://github.com/bochecha/dbus-python.git@pipified-1.1.1#egg=dbus
```

Comment:
Also, I'd like to be sure we have made everything we can for this to be solved asap: ticket on Travis so we can use a newer version, ticket on the github dbus fork so they merge our changes, ticket on the upstream dbus repo so they release on pypi, etc. :)
We will not wait for anything, but I just want to help pushing the community forward on those topic so the situation may get cleaner at some point :)